### DISCUSSION TITLE
plot dynamic layer

### DISCUSSION THREAD

**Discussion by rh-afk** [opengeos/leafmap#520] [Q&A] [answered]
Hi all, kindly please advice how to plot dynamic layer. I try use this one but no layer comes up:
![image](https://github.com/opengeos/leafmap/assets/141318843/a7d50a91-694e-48d8-9b0d-5fb535801e21)

expected result should be like in here:
[https://developers.arcgis.com/esri-leaflet/samples/simple-dynamic-map-layer/](url)

thanks

**Comment by giswqs** [comment, marked as answer]
Not all tile layers are supported. 

```python
m = leafmap.Map()
url = 'https://server.arcgisonline.com/ArcGIS/rest/services/World_Street_Map/MapServer/tile/{z}/{y}/{x}'
m.add_tile_layer(url, name='ArcGIS', attribution='Esri')
m
```

**Reply by rh-afk** [reply]
thanks @giswqs , means dynamic map tile layer not supported in leafmap?

**Reply by giswqs** [reply]
Try ipyleaflet. Leafamp support anything ipyleaflet supports. If ipyleaflet does not support it, leafmap won't work either.
